tests/integrationv2/configuration.py

```diff
@@ -20,7 +20,7 @@
 
 
 # List of providers that will be tested.
-PROVIDERS = [S2N, OpenSSL, JavaSSL]
+PROVIDERS = [CriterionS2N, S2N, OpenSSL, JavaSSL]
```

Comment:
Since you have to check for S2N_USECRITERION and swap S2N with CriterionS2N, do you ever actually want both of them in the PROVIDERS list?

Reply:
Possibly not, let me review where PROVIDERS is being used.

Reply:
Removed.